sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1981,6 +1991,32 @@ public void onCaseChanged(CaseDataDto existingCase, Case newCase, boolean syncSh
 		}
 	}
 
+	public boolean evaluateFulfilledCondition(CaseDataDto newCase, CaseClassification caseClassification) {
+
+		if (newCase.getCaseClassification() != CaseClassification.NO_CASE) {
```

Comment:
This if condition is not necessary because the code in the if block will return false for a case classified as NO_CASE anyway, right?

Reply:
The code in the if block might return Confirmed. The condition is that if NO_CASE is set the Reference Definition will be always NOT_FULFILLED